README.md

```diff
@@ -25,12 +25,25 @@ pnpm add @unleash/nextjs
 This package will attempt to load configuration from
 [Next.js Environment variables](https://nextjs.org/docs/basic-features/environment-variables).
 
-| Prefixable     | Variable                     | Default                                                   | Used in                                              |
-| -------------- | ---------------------------- | --------------------------------------------------------- | ---------------------------------------------------- |
-| `NEXT_PUBLIC_` | `UNLEASH_BASE_URL`           | `http://localhost:4242/api`                               | `FlagProvider`, `getFrontendFlags`                   |
-| **No**         | `UNLEASH_API_TOKEN`          | `default:development.unleash-insecure-api-token`          | `getDefinitions`                                     |
-| `NEXT_PUBLIC_` | `UNLEASH_FRONTEND_API_TOKEN` | `default:development.unleash-insecure-frontend-api-token` | `FlagProvider`, `getFrontendFlags`                   |
-| `NEXT_PUBLIC_` | `UNLEASH_APP_NAME`           | `nextjs`                                                  | `FlagProvider`, `getFrontendFlags`, `getDefinitions` |
+When using Unleash **client-side**, with `<FlagProvider />` or `getFrontendFlags()` configure:
+
+  - `NEXT_PUBLIC_UNLEASH_FRONTEND_API_URL`. URL should end with `/api/frontend` or `/proxy`
+  - `NEXT_PUBLIC_UNLEASH_FRONTEND_API_TOKEN` [client-side Unleash token](https://docs.getunleash.io/reference/api-tokens-and-client-keys#front-end-tokens)
```

Comment:
If you're using the proxy, won't this be a proxy client key instead?

Something like this, maybe?

```suggestion
  - `NEXT_PUBLIC_UNLEASH_FRONTEND_API_TOKEN` [client-side API token](https://docs.getunleash.io/reference/api-tokens-and-client-keys#front-end-tokens) if you're using the front-end API, or a [proxy client key](https://docs.getunleash.io/reference/api-tokens-and-client-keys#proxy-client-keys) if you're using a proxy
```